### ISSUE TITLE
Add a Nomad Solution on Dashboard

### ISSUE THREAD

**Issue by mik-tf** [threefoldtech/tfgrid-sdk-ts#3758]
### Which package/s are you suggesting this feature for?

Dashboard

### Is your feature request related to a problem? Please describe

As discussed with Thabet and Scott, it would be good to add a Nomad solution on the Dashboard. 

### References

- The manual already covers a basic manual guide for Nomad: https://manual.grid.tf/documentation/system_administrators/terraform/advanced/terraform_nomad.html#deploy-a-nomad-cluster

### Describe the solution you'd like

We propose to add a nomad solution on the dashboard allowing to deploy multiple server and client nodes.

### Proposition of Configurations

- Minimum: 3 server nodes, 3 client nodes
- Basic: 3 server nodes, 3-5 client nodes
- Advanced: 5 server nodes, 5-20 client nodes

The basic configuration would be a good start.

### Link with GlusterFS + CSI

I think we can add container storage interface ([CSI](https://github.com/container-storage-interface/spec)) with GlusterFS to allow for HA for app (nomad) and HA for storage (glusterFS). Might be for a separate issue once GlusterFS is done on Dashboard too (feature request: https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3759)

**Comment by mik-tf** [comment]
As discussed with Thabet and Kristof, we will first start by having guides instead of dashboard solutions.
